application/go_inc/helpers/helper__functions.php

```diff
@@ -78,7 +78,17 @@ function go_get_menu_items($menu_id) {
         foreach($query->result() as $result){
             if($count == 1) {
                 $return .= "<div id='menu-id-" . $result->MenuID . "'>"; // setup parent DIV and UL
-                $return .= "<ul>"; 
+
+                //$return .= "<ul>"; 
+                
+                if($menu_id == 1){
```

Comment:
Let's leave this how you have it for now.  When it comes time to have more than 2 menus in Admin, we will address it with larger effort.  This may require some refactor by then anyways.   This is fine.